sdk/examples/c2pa.toml

```diff
@@ -90,7 +90,7 @@ action = "c2pa.edited"
 # The source type field is required for the c2pa.created action.
 #
 # For more information, see `c2pa::assertions::actions::source_type`.
-source_type = "http://c2pa.org/digitalsourcetype/empty"
+source_type = "digsrctype:empty"
 # Description for the action
```

Comment:
where is this defined? shouldn't it be DigitalSourceType::Empty?

Reply:
Every digital source type has an "ID (QCode)" so I figured it would make more sense to use that.

<img width="773" height="144" alt="Screenshot 2025-07-30 at 9 48 33 AM" src="https://github.com/user-attachments/assets/688f3a2c-3907-4378-84ac-e17ac962c1a7" />

Reply:
Yes.